packages/kotti-ui/source/next/kotti-field-date/mixins.scss

```diff
@@ -226,10 +226,14 @@
 								}
 
 								&:not(.disabled) {
-									@supports (
-										clip-path: circle(calc(#{$td-diameter/2} + 2px) at center)
-									) {
-										clip-path: circle(calc(#{$td-diameter/2} + 2px) at center);
+									&:not(.in-range) {
```

Comment:
but this was the style for the selected dates (end and start) which have an `.in-range` class 
if we exclude in-range then i exclude the start and end date from having a circle clip path

Reply:
Tested this and they’re still round, so I’m resolving.